Source/WebKit/UIProcess/Cocoa/UIDelegate.mm

```diff
@@ -447,6 +447,13 @@
         return;
     }
 
+    if (organizationStorageAccessPromptQuirk) {
+#if !PLATFORM(WATCHOS) && !PLATFORM(APPLETV)
```

Comment:
This should probably enclose the whole if and return.
Also, can we give it a name, like #if !HAVE(whatever_is_needed_here)

Reply:
I was following the existing pattern in that function, seems like we can clean that up a bit.